components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -4154,4 +4154,5 @@ public static String getProcessedClaimValue(String claimURI, String claimValue,
             throw new FrameworkException("Error occurred while encrypting claim value of: " + claimURI, e);
         }
     }
+
```

Comment:
shall we revert this change as its irrelevant?

Reply:
Fixed a0f882011c467a8f0c781821bfcd6ad8f03da0c5